AirSkyBoat/AirSkyBoat
# fixed a few places where KIs not properly gives to players

**_I affirm:_**

## What does this pull request do?

As part of researching #1462 , I discovered there were a few more places where the KI function wasn't being called properly.

## Steps to test these changes

Run through the following quests and ensure KIs are awarded as expected:
Dungeons and Dancers
Curses Foiled A Golem
Flapano
